UWP/HockeyAppDemoUwp/cs/App.xaml.cs

```diff
@@ -31,20 +31,33 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
-            HockeyClient.Current.Configure(DemoConstants.YOUR_APP_ID,
-                new TelemetryConfiguration() { EnableDiagnostics = true})
-                .SetContactInfo("DemoUser", "demoapp@hotmail.com")
-                .SetExceptionDescriptionLoader((Exception ex) =>
-                {
-                    return "Exception HResult: " + ex.HResult.ToString();
-                });
-
-            // Microsoft.HockeyApp.HockeyClient.Current.Configure(DemoConstants.YOUR_APP_ID);
-
+            // If you don't configure Hockey App in your App constructor, you need to configure it
+            // at all entry points to the application: Activated, Launched, and Resuming (in the
+            // case of a prelaunch, in which we deliberately don't configure Hockey App).
+            //
+            // The motivation behind not configuring HockeyApp in the App constructor is to avoid
+            // a session start during a prelaunch. If you do not care about this in your app, you
+            // may remove all calls to ConfigureHockeyApp and configure it in your App constructor.
+ 
+            this.Resuming += App_Resuming;
             this.InitializeComponent();
             this.Construct();
         }
 
+        private void App_Resuming(object sender, object e)
+        {
+            // If an app is opened after being prelaunched, OnLaunched is not called and
+            // the Resuming event gets fired.
+            ConfigureHockeyApp();
```

Comment:
Is there a scenario where App_Resuming is called and OnLaunched or OnActivated are not?